src/main/java/com/griddynamics/cloud/learning/web/dto/ErrorResponse.java

```diff
@@ -4,9 +4,13 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.validation.constraints.NotBlank;
+
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class ErrorResponse {
+
+    @NotBlank
```

Comment:
Please, use such stuff consciously.
It's dedicated for requests' payloads, that are gonna be validated